core/src/main/java/org/jboss/pnc/rex/facade/mapper/GraphsMapper.java

```diff
@@ -56,4 +61,60 @@ default void applyCorrelationID(CreateGraphRequest source, @MappingTarget TaskGr
             target.vertices(vertices);
         }
     }
+
+    @BeforeMapping
+    default void mergeWithGraphConfig(CreateGraphRequest request) {
```

Comment:
Nice merging general graph and task specific configuration! :+1:

Reply:
❤️